src/app/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -0,0 +1,155 @@
+import dayjs from 'dayjs';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import utc from 'dayjs/plugin/utc';
+import moment from 'moment';
+import { MutableRefObject, useMemo, useRef } from 'react';
+import 'react-big-calendar/lib/sass/styles.scss';
+import { Calendar, momentLocalizer, Event, EventProps } from 'react-big-calendar';
+import { RRule } from 'rrule';
+
+import { CalendarEvent } from './CalendarEvent';
+
+dayjs.extend(utc);
+dayjs.extend(customParseFormat);
+
+const localizer = momentLocalizer(moment);
+
+const eventStyleGetter = ({ resource }: Event) => {
+  var style = {
```

Comment:
If we want to keep a `style` object assignment here, we should have this be a `const`.

Reply:
I'll change it to a const.